### ISSUE TITLE
Block cages remain when server restarts

### ISSUE THREAD

**Issue by Muqsit** [Muqsit/SkyWars#11]
With block cages enabled, if you are in a game that hasn't yet started and you restart the server, the block cages will remain there instead of being deleted during `SkyWars::reset()` call.

**Comment by FreeGamingHere** [comment]
Foreach all the players inside the arenas when the plugin is getting disabled. Then check if the blocks near them (yes you have to check one by one) are glasses. If it returns true you set them as air blocks. This is a temp solution because it may not be a good idea for the servers that are getting killed.

**Comment by Muqsit** [comment]
@FreeGamingHere yeah, that's what is happening over here https://github.com/Muqsit/SkyWars/blob/master/src/muqsit/skywars/Loader.php#L131

**Comment by FreeGamingHere** [comment]
Uhh...

**Comment by xXNiceYT** [comment]
Is bug fixed?
